scripts/clean-shrinkwrap.js

```diff
@@ -25,7 +25,7 @@ const EXIT_CODES = require('../lib/shared/exit-codes');
 const SHRINKWRAP_PATH = path.join(__dirname, '..', 'npm-shrinkwrap.json');
 
 try {
-  console.log(childProcess.execSync('npm shrinkwrap', {
+  console.log(childProcess.execSync('npm shrinkwrap --dev', {
     cwd: path.dirname(SHRINKWRAP_PATH)
   }));
```

Comment:
I think you need to change this to `  }).toString());` ? Otherwise I just get `<Buffer` and a load of hex-digits printed to the screen.

And I think this script might need to be using `npm prune` too? In fact, could the `CONTRIBUTING.md` instructions be updated to use _only_ this script (even if that means modifications are needed to this script also), as currently it's unclear in the instructions when to use `npm run clean-shrinkwrap` and when to use `npm shrinkwrap --dev`.